### PR TITLE
Add styling for Alerts page

### PR DESCRIPTION
**Pull request by shandyk18**
- Add styling for Alerts page
  - Add hover styling on each card
- Add primary and secondary text color palette in global theme
- Separate Time Picker component from AlertsComponent

image

### CODE REVIEW

File: src/components/pages/alerts/Alert.jsx [hunk 1]
```diff
@@ -1,63 +1,85 @@
...
-            <Box marginRight="1vw">
+         <CardContent
+            onMouseOver={() => setBellIcon(<BellIcon />)}
+            onMouseOut={() => setBellIcon(<BellIcon2 />)}
+            className={classes.cardContent}
+         >
+            <Box alignSelf="flex-start">
                <IconButton className={classes.bellIcon}>
-                  <BellIcon />
+                  {bellIcon}
```
Comment (Dat-Hoang-Vo) [inline src/components/pages/alerts/Alert.jsx:66]: Does the B in "bellIcon" need to be capitalized? You imported it as a capitalized B.

File: src/components/pages/alerts/Alert.jsx [hunk 2]
```diff
@@ -1,63 +1,85 @@
...
+               <Box>
+                  <Typography variant="subtitle2" color="textSecondary">OCTOBER 4, 2021</Typography>
+               </Box>
+               <Box marginBottom="1vh">
+                  <Typography variant="h6" color="textPrimary">AS3 Reading is currently very unhealthy</Typography>
+                  <Typography variant="h6" color="textPrimary" gutterBottom="true">Open window in Room 3</Typography>
                </Box>
                <Box>
-                  <Typography>Sensor message</Typography>
+                  <Typography variant="body1" color="textPrimary">High carbon dioxide levels detected</Typography>
```
Comment (Dat-Hoang-Vo) [inline src/components/pages/alerts/Alert.jsx:78]: The typography doesn't have a close.

**Review by Dat-Hoang-Vo — CHANGES_REQUESTED** [review]

File: src/components/pages/alerts/Alert.jsx (hunk 1 above)
Comment (shandyk18) [reply src/components/pages/alerts/Alert.jsx:66]: bellIcon is a state that can either be BellIcon or BellIcon2. The onMouseOver in CardContent sets bellIcon to be either icons, depending on if the alert is hovered over or not.

File: src/components/pages/alerts/Alert.jsx (hunk 2 above)
Comment (shandyk18) [reply src/components/pages/alerts/Alert.jsx:78]: Sorry I don't think I understand this comment. Do you mean the closing tag?

File: src/components/pages/alerts/Alert.jsx (hunk 1 above)
Comment (Dat-Hoang-Vo) [reply src/components/pages/alerts/Alert.jsx:66]: Oh okay sounds good

File: src/components/pages/alerts/Alert.jsx (hunk 2 above)
Comment (Dat-Hoang-Vo) [reply src/components/pages/alerts/Alert.jsx:78]: Nvm I'm blind I see it now. Ignore my comment LOL

**Review by Dat-Hoang-Vo — APPROVED** [review]

**Pull request merged**